shared/src/packages/auth/libs/enums/auth-api-path.enum.ts

```diff
@@ -1,6 +1,7 @@
 const AuthApiPath = {
   ROOT: '/',
   SIGN_UP: '/sign-up',
+  USER: '/authenticated-user',
```

Comment:
key and value should be the same

btw, usually this route is named as `/current`

Reply:
please fix this comment) you should rename USER key to AUTHENTICATED_USER